client/packages/system/src/Item/Components/ItemVariantSearchInput/ItemVariantSearchInput.tsx

```diff
@@ -30,7 +30,7 @@ export const ItemVariantSearchInput = ({
     <Autocomplete
       disabled={disabled}
       width={`${width}px`}
-      popperMinWidth={Number(width)}
+      popperMinWidth={200}
```

Comment:
Was there a reason you changed this? Now getting this in bundled items:

![Screenshot 2024-11-26 at 11 26 45 AM](https://github.com/user-attachments/assets/c1087e20-f41c-4494-b340-9932085d8fde)

But I think it was working ok? Popper expands to whatever width is needed to show the variant names?

Reply:
Dang, this was based on Mark's suggestion here: https://github.com/msupply-foundation/open-msupply/issues/5514#issuecomment-2496256771

Reply:
Forgot to Comment with the reference, I'll do a min of the two maybe?

Reply:
<img width="945" alt="image" src="https://github.com/user-attachments/assets/1557ad7f-c05b-40ab-b467-97112c71f617">